### PR TITLE
Jp jukebox activesong

### PR DESCRIPTION
**Pull request by JP5457**
Current jukebox playlist display on timelord

### CODE REVIEW

File: src/Classes/ServiceAPI/MyRadio_Timeslot.php [hunk 1]
```diff
@@ -687,6 +688,8 @@ public static function getCurrentAndNext($time = null, $n = 1, $filter = [1])
         $timeslot = self::getCurrentTimeslot($time, $filter);
         $next = self::getNextTimeslot($time, $filter);
 
+        $currentJukebox = iTones_playlist::getPlaylistAtTime(date('H:i:s'));
```
Comment (markspolakovs) [inline src/Classes/ServiceAPI/MyRadio_Timeslot.php:691]:
```suggestion
        $currentJukebox = iTones_Playlist::getPlaylistAtTime(date('H:i:s'));
```

File: src/Classes/iTones/iTones_Playlist.php [hunk 2]
```diff
@@ -235,6 +235,27 @@ public function getTitle()
         return $this->title;
     }
 
+    public function getPlaylistAtTime($time)
+    {
+        $getplaylist = self::$db->fetchColumn(
+        'SELECT playlist.playlistid
+        FROM jukebox.playlists playlist
+        INNER JOIN jukebox.playlist_availability avail ON playlist.playlistid=avail.playlistid
+        WHERE playlist.category=2
```
Comment (markspolakovs) [inline src/Classes/iTones/iTones_Playlist.php:244]: Do we have a `Config` variable for the category ID? We'd rather not hard-code it if possible.

File: src/Classes/iTones/iTones_Playlist.php [hunk 3]
```diff
@@ -235,6 +235,27 @@ public function getTitle()
         return $this->title;
     }
 
+    public function getPlaylistAtTime($time)
```
Comment (markspolakovs) [inline src/Classes/iTones/iTones_Playlist.php:238]: What will this do if there are multiple active playlists at a time?

Comment (JP5457) [reply src/Classes/iTones/iTones_Playlist.php:238]: just return one of them. This shouldn't matter as we are planning to move to only having a single active playlist at any point.

**Pull request closed without merging**